lib/cleanupDone.js

```diff
@@ -0,0 +1,17 @@
+function cleanupDone(config, sftp) {
+  var root = config.fileDownloadDir + '/' + config.completedDir
+  return sftp.list(root)
+    .then(files => {
+      return Promise.all(files.filter(file => {
+        const age = Date.now() - file.modifyTime
+        const keep = 1000 * config.fileRetentionDays * 60 * 60 * 24
```

Comment:
Does `config.fileRetentionDays` have a default value? What happens if it's not set?

Reply:
Not sure actually, other config values probably have the same problem.